clients/drcachesim/tests/drmemtrace.threadsig.aarch64.raw/README

```diff
@@ -0,0 +1,50 @@
+How to re-generate the drmemtrace.threadsig.aarch64.raw files
+
+The application run is "threadsig" from https://github.com/DynamoRIO/drmemtrace_samples/blob/master/threadsig.aarch64/threadsig.cp
+
+To save repository space, build it with -Os and -static on an AArch64
+machine or cross-compile it:
+
+  $ g++ -o threadsig -Os threadsig.cpp -lpthread -static
+  $ strip threadsig
+
+Now run it with 4 threads and 2000 iterations, and possibly
+-unsafe_build_ldstex depending on your architecture:
+
+  $ bin64/drrun -unsafe_build_ldstex -t drcachesim -offline -- ~/threadsig 4 2000
+
+Compress the raw files:
+
+  $ gzip `ls -1td drmemtrace.threadsig.*.dir|head -1`/raw/*.raw
+
+Now copy the new data files and the threadsig and libdynamorio.so.
+
+Strip libdynamorio.so, though this may not decrease it much:
+
+  $ strip lib64/release/libdynamorio.so
+
+There is no need to change the checked-in libdrmemtrace.so.  It is not
```

Comment:
Curious why dynamorio includes it in the list of modules if it is not executed?

Reply:
You mean, why is a client library in the list of app libraries?  And why just the top-level client and none of the extension libraries it uses?  Some client libraries do have the app execute some of their code (such as when using drwrap_replace).  The client lib is mapped early, and its deps loaded later, and the memory walk finds it, and we deemed it better to include it than omit it for the code reason.  There is an issue on it IIRC.